packages/app/source/server/projections/open-appeals-projection.js

```diff
@@ -0,0 +1,36 @@
+Space.eventSourcing.Projection.extend(Donations, 'OpenAppealsProjection', {
```

Comment:
The public-facing appeals view will contain less data and ideally we'd have separate apps making this highly optimised. Is it not a good move for us to be building in preparation for that design given there's really no impact?